src/lexer.coffee

```diff
@@ -121,7 +121,11 @@ exports.Lexer = class Lexer
         'PROPERTY'
       else
         'IDENTIFIER'
-
+    
+    if tag is 'IDENTIFIER' and @seenFor and id is 'from'
```

Comment:
`from` is not really a relation like `in`, `of` or `instanceof`. It can't be used as an infix operator. You can do `x in y` and `x of y` but not `x from y`.